components/dashboard/src/service/json-rpc-organization-client.ts

```diff
@@ -249,7 +251,10 @@ export class JsonRpcOrganizationClient implements PromiseClient<typeof Organizat
                 "updateRestrictedEditorNames is required to be true to update restrictedEditorNames",
             );
         }
-        await getGitpodService().server.updateOrgSettings(request.organizationId, update);
+        await getGitpodService().server.updateOrgSettings(request.organizationId, {
+            ...update,
+            defaultRole: request.defaultRole as OrgMemberRole,
```

Comment:
This line seems superfluous :monocle_face:

Reply:
it casts string to OrgMemberRole

Reply:
Ah, different types, yummy :grimacing: